src/ott/tools/sliced.py

```diff
@@ -88,10 +88,15 @@ def sliced_wasserstein(
   Returns:
     The sliced Wasserstein distance with the corresponding output object.
   """
-  if proj_fn is None:
+  if projector is None:
```

Comment:
I think this is a bit complex, no? it just saves the user ``lambda arr, **_: arr @ projector.T`` when passing ``proj_fn`` as matrix product, at the cost of this ambiguity

My hunch is that most users will likely want to change the way directions are sampled in the proj, rather than pass a matrix (that was the idea of passing a random generator first), or, alternatively, define a proper ``proj_fn`` feature extractor (that's also why I had added the `features` word above, as, when the ``proj_fn`` becomes more complex, one should rather see it as a feature extractor)